volttron/platform/vip/agent/subsystems/ping.py

```diff
@@ -82,6 +82,8 @@ def ping(self, peer, *args):
         result = next(self._results)
         args = list(args)
         args.insert(0, b'ping')
+        if peer == b'pubsub':
+            peer = b''
```

Comment:
So b'' is the vip_identity of the router.  Is pubsub not addressable using the vip identity pubsub?  I suppose not since it is part of the router.  We need to put pubsub up a level at self.vip.publish(topic, message, headers=None) rather than under the self.vip.pubsub.  Because now the pubsub is useless in the call.